tests/data/config/w.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+# config now can have imported modules and defined functions for convenience
+import os.path as osp
+def func():
+    return 'string with \"escape\" characters'
```

Comment:
It seems `'string with \"escape\" characters' == 'string with "escape" characters'` is True and str_items2 maybe not with an escape character?

Reply:
Yes, python string need no escape for quotes. I'll change to another example.